src/Data/Bin/Base.agda

```diff
@@ -0,0 +1,109 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Arithmetic of binary represented natural numbers. Initial part.
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Data.Bin.Base where
+
+open import Algebra.FunctionProperties using (Op₂)
+open import Data.Nat        using (ℕ) renaming (suc to 1+_; _*_ to _*ₙ_)
+open import Function        using (_∘_)
```

Comment:
You probably don't need to import more than `Data.Nat.Base`.

Reply:
The whole import changed to
```
  open import Algebra.FunctionProperties using (Op₂)
  open import Data.Nat.Base as ℕ using (ℕ)
```